corehq/apps/reminders/views.py

```diff
@@ -492,7 +492,7 @@ class AddStructuredKeywordView(BaseMessagingSectionView):
     template_name = 'reminders/keyword.html'
     process_structured_message = True
 
-    @method_decorator(requires_privilege_with_fallback(privileges.OUTBOUND_SMS))
+    @method_decorator(requires_privilege_with_fallback(privileges.INBOUND_SMS))
```

Comment:
@orangejenny it was actually wrong, these views shouldn't be seen unless the project has access to inbound sms